rita_exit/src/database/mod.rs

```diff
@@ -370,20 +371,24 @@ pub fn cleanup_exit_clients(
 pub fn setup_clients(
     clients_list: &[exit_db::models::Client],
     old_clients: &HashSet<ExitClient>,
+    setup_wg_exit: &mut HashSet<WgKey>,
+    setup_wg_exit_v2: &mut HashSet<WgKey>,
```

Comment:
return pattern instead of mutable pointers here as well.

Reply:
done